Machine Learning/Analytics Jupyter Notebooks/ml-analytics-classification.ipynb

```diff
@@ -0,0 +1,469 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Predicting delayed flights with classification analysis\n",
+    "\n",
+    "[This is a companion Jupyter notebook to the documentation example.](https://www.elastic.co/guide/en/elastic-stack-overview/7.5/flightdata-classification.html)\n",
+    "\n",
+    "Let’s try to predict whether a flight will be delayed or not by using the sample flight data. We want to be able to use information such as weather conditions, carrier, flight distance, origin, or destination to predict flight delays. There are only two possible outcome values: the flight is either delayed or not, therefore we use binary classification to make the prediction.\n",
```

Comment:
It might be worth mentioning that we only support binary at this stage.

Reply:
Thank youuuu! this is a good point - will add